### DISCUSSION TITLE
如果antd继续升级，formily会怎么处理？

### DISCUSSION THREAD

**Discussion by wjwjwjw** [alibaba/formily#1410] [Ideas]
如果antd继续升级，formily会怎么处理？formily后续的发展计划是怎么样的？

**Comment by janryWang** [comment]
好问题，如果antd发生较大break change，那formily整体项目也升大版本来处理，如果没有br，但是又有较大更新，那formily会做版本兼容

**Reply by wjwjwjw** [reply]
那对于2.0有还有什么开发计划么，会有新的功能支持，还是主要是对现有功能进行维护？

**Reply by janryWang** [reply]
表单设计器是计划内的，其他是完善Vue组件生态，剩下的是明后年的事情了，先不考虑
